packages/js/admin-layout/src/gutenberg/admin-layout.tsx

```diff
@@ -8,10 +8,9 @@ import {
 } from '@wordpress/compose';
 import { __ } from '@wordpress/i18n';
 import {
-	// @ts-expect-error missing type.
 	EditorSnackbars,
-	// @ts-expect-error missing type.
 	privateApis as editorPrivateApis,
+	// @ts-expect-error missing type.
```

Comment:
A generic wp-admin screen, like the one for Woo Settings Editor, shouldn't import from `@wordpress/editor`. There isn't any block editor on that screen, is there?

The `editorPrivateApis` are used to access `NavigableRegion`, but that's a re-export of the `NavigableRegion` from `@wordpress/interface`. I'd say that the admin screen should use and bundle the `@wordpress/interface` package directly, that's how the package is supposed to be used.

The `editor` re-export is there only for the `edit-post` and `edit-site` packages so that they can share the `interface` store with `editor`. Added in #60748 by @youknowriad and @oandregal.

The `EditorSnackbars` component is not specific to `editor` at all and should probably live somewhere else. It's a very simple combination of the `SnackbarList` from `@wordpress/components` and the `notices` store from `@wordpress/notices`. For WooCommerce, you should probably reimplement it yourself using these two low-level general-purpose packages.